src/worker/queue.ts

```diff
@@ -66,26 +67,22 @@ function startQueueRedis(server: PluginsServer, piscina: Piscina | undefined, wo
             now: string,
             sent_at?: string
         ) => {
-            const event = { distinct_id, ip, site_url, team_id, now, sent_at, ...data } as PluginEvent
+            const event = {
+                distinct_id,
+                ip,
+                site_url,
+                team_id,
+                now,
+                sent_at,
+                uuid: new UUIDT().toString(),
```

Comment:
Why `uuid` added here?

Reply:
Because of this: https://github.com/PostHog/plugin-server/blob/22c49ba019cfb5b093e907ccb6883ef281cd0e3d/src/ingestion/process-event.ts#L59-L61

Even though we don't use it when there's no kafka/clickhouse, without it plugin server ingestion was broken on OSS :). I could add a `&& kafka` to the check, but that would require `Optional`-ing a lot of types and adding many more `if no uuid` checks into inner methods...

Reply:
Alright